src/OrchardCore/OrchardCore.FileStorage.FileSystem/OrchardCore.FileStorage.FileSystem.csproj

```diff
@@ -0,0 +1,11 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard1.6</TargetFramework>
```

Comment:
1.6?

Reply:
Didn't change this, this is a rename of an existing project. But I can update to 2.0 if that's what we're targeting by default for all libraries now?